providers/webex-chat/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@novu/webex-chat",
+  "version": "0.16.3",
```

Comment:
This should be the latest version 0.20 something

Reply:
This is the most recent version.